dags/chronos_train_dag.py

```
@@ -0,0 +1,344 @@
+
+        import torch
+        torch.set_default_device('cpu')
+        if hasattr(torch.backends, 'mps'):
+            torch.backends.mps.is_available = lambda: False
+            torch.backends.mps.is_built = lambda: False
+
+        import mlflow
+        from autogluon.timeseries import TimeSeriesDataFrame
+        import shutil
+
+        MODEL_DIR = '/tmp/autogluon_models'
```

Ruff (0.12.2)

Probable insecure usage of temporary file or directory: "/tmp/autogluon_models"

(S108)